app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -151,10 +154,9 @@ def save_challenge_in_session
 
     def process_valid_webauthn(form)
       create_user_event(:webauthn_key_added)
-      mfa_user = MfaContext.new(current_user)
-      analytics.multi_factor_auth_added_webauthn(
```

Comment:
Since this was the only use of `multi_factor_auth_added_webauthn`, can we remove the method definition from `analytics_events.rb` ?

Reply:
yup, just removed this event